app/dashboards/article_dashboard.rb

```diff
@@ -61,7 +61,7 @@ class ArticleDashboard < Administrate::BaseDashboard
   # Overwrite this method to customize how articles are displayed
   # across all pages of the admin dashboard.
   #
-  # def display_resource(article)
-  #   "Article ##{article.id}"
-  # end
+  def display_resource(article)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)